lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -600,6 +616,12 @@ def ach?(payment_method)
           card_brand(payment_method) == "check"
         end
       end
+
+      def auth_minimum_amount(options)
+        return 100 unless options[:currency]
+        return 100 unless MINIMUM_AUTHORIZE_AMOUNTS.has_key?(options[:currency].upcase)
+        return MINIMUM_AUTHORIZE_AMOUNTS[options[:currency].upcase]
+      end
```

Comment:
Little tweak to simplify?

```
def auth_minimum_amount(options)
  return 100 unless options[:currency]
  MINIMUM_AUTHORIZE_AMOUNTS[options[:currency].upcase] || 100
end
```

Not a biggie though.  😄 

<img width="364" alt="ship_little_-_google_search" src="https://cloud.githubusercontent.com/assets/2437/19054641/48cff646-898e-11e6-8a43-9684b4c395e3.png">